backend/models/user.js

```diff
@@ -23,6 +23,27 @@ const UserSchema = new Schema({
     type: String,
     required: true,
   },
+  phone: {
+    type: String,
+    required: true,
+  },
+  github_username: {
+    type: String,
+    required: true,
+  },
+  discord_username: {
+    type: String,
+    required: true,
+  },
+  linkedin_username: {
+    type: String,
+    required: true,
+  },
```

Comment:
Do you think that these should be `required: false` in case they don't have these accounts? Or should we require users to have them

Reply:
Oh yes, good point, I think it should be `required: false`